src/app/shared/example-viewer/example-viewer.html

```diff
@@ -3,6 +3,8 @@
     <div class="docs-example-viewer-title-spacer">{{exampleData?.title}}</div>
 
     <button md-icon-button type="button" (click)="toggleSourceView()"
+            role="button"
```

Comment:
`role="button"` should never be necessary on a `<button>` element

(here and elsewhere)